src/register.ts

```diff
@@ -23,3 +24,9 @@ Module.prototype._compile = function _compile(code: string, fileName: string): s
 Module.prototype.require = new Proxy(Module.prototype.require, {
   apply: requireHook,
 });
+
+declare global {
+  // eslint-disable-next-line no-var
+  var AppMapRecordHook: typeof recordHook;
+}
+global.AppMapRecordHook = recordHook;
```

Comment:
If the whole machinery in `hooks/index.ts` is now unused you can remove it.

Reply:
Only jest hook (`hooks/jest.ts`) is using it for now. I can change it as well, if it's ok for you. Then I can remove the old machinery.

Reply:
Ah, ok. Let's leave it for now in that case.